src/sunadaptcontroller/imexgus/fmod_int32/CMakeLists.txt

```diff
@@ -17,10 +17,8 @@ sundials_add_f2003_library(sundials_fsunadaptcontrollerimexgus_mod
     fsunadaptcontroller_imexgus_mod.f90 fsunadaptcontroller_imexgus_mod.c
   LINK_LIBRARIES
     PUBLIC sundials_fcore_mod
-  OBJECT_LIBRARIES
```

Comment:
I think we should keep the controllers as object libraries only since they're already part of ARKODE and not very useful as a standalone library.